test/validate-reviews.test.js

```diff
@@ -0,0 +1,18 @@
+import app from "../server/server";
+import { describe, expect, test } from "@jest/globals";
+import validateReviews from "../server/validateReviews";                                                           
+
+describe("Test review validation", () => {
+    test("rating should be integer between 0 and 5", async () => {
+        const review = {
+            
+            author: 'kalle',
+            rating: 3,
+            comment: "medelmåttigt"
+        }
+    })
+        const result = validateReviews((review,res) =>{
+            
+		expect(res.status===200);
+	});
```

Comment:
These three lines (four including the blank line) execute the `validateReviews` function, passing an anonymous function as it's only parameter. However, the `validateReviews` function doesn't do anything with it's parameter, so this code effectively does nothing. The anonymous function contains an `expect()` but will never be called.